lib/version.js

```diff
@@ -1,3 +1,4 @@
-"use strict";
 
-module.exports = "1.1.0";
+'use strict';
```

Comment:
version file should only get touched as a part of the release process.  please revert this one.

Reply:
your test set enforces that the build number is correct as part of the build process

if the local build can't update it then that local test will always fail

Reply:
it's strongly undesirable for the local build and the remote build to not match behaviorally

Reply:
Hm.  I thought I had removed that test so that the web builds wouldn't need to include package.json, but that must have been in the patch that didn't land.

I don't want the version number to change until we make an explicit decision to cut a release.  However, I don't understand your full proposed release flow yet, so I'm willing to wait until I see the big picture if you think I'll agree later.

Reply:
Okay so in reading through this I realized there were two ways to read the commentary being made, and I'm not sure whether we're talking about one or both

1. Should John have set 1.2.0?  No.  That's the job at the final step.  This is a mistake on my part which is being undone.
2. Should there be infrastructural software handling the propogation of the number from the package file into the source?  That is what I was arguing in favor of.

If we're discussing whether the build system should work this way, my opinion is that it shouldn't change.

If we're discussing whether I jumped the gun on declaring a version, yes, I did this by habit and am undoing it.

I would appreciate if further discussion could clarify which of those, or something else, is meant in context